planex/cmd/makesrpm.py

```diff
@@ -240,11 +248,14 @@ def main(argv=None):
 
     args = parse_args_or_exit(argv)
     tmpdir = tempfile.mkdtemp(prefix="px-srpm-")
+    for subdir in [ 'BUILD', 'BUILDROOT','RPMS','SOURCES','SPECS','SRPMS' ]:
+        os.mkdir(os.path.join(tmpdir, subdir))
 
     try:
         specfile = populate_working_directory(tmpdir, args.spec, args.link,
                                               args.sources, args.patchdata)
-        sys.exit(rpmbuild(args, tmpdir, specfile))
+        if rpmbuild(args, tmpdir, specfile) == 0:
```

Comment:
If `rpmbuild()` returns non-zero will `makesrpm`'s exit code now reflect that?

Reply:
Point. Have updated.